src/script/components/list/ConversationListCell.tsx

```diff
@@ -94,7 +93,7 @@ const ConversationListCell: React.FC<ConversationListCellProps> = ({
   const cellBottom = cellTop + cellHeight;
 
   const isInitiallyVisible = isVisibleFunc(cellTop, cellBottom);
-  const [viewportElementRef, setViewportElementRef] = useEffectRef<HTMLDivElement>();
+  const [viewportElementRef, setViewportElementRef] = useEffectRef<HTMLElement>();
```

Comment:
why not `HTMLLiElement`?

Reply:
It's because I don't need that level of specificity. Would work with any kind of element `div` `li` or any other.

Reply:
ahh, ok - it is used by the `useViewPortObserver` hook - it makes sense